### PR TITLE
Developer 1178 Added BPM Suite starter-kit to BPM Suite getting-started page

### PR DESCRIPTION
**Pull request by adelasofia**

### CODE REVIEW

**Comment by paulrobinson** [comment]
this is ok to test

File: _partials/bpm-suite-starter-kit.html.slim [hunk 1]
```diff
@@ -0,0 +1,121 @@
+.row
+  .large-24.columns
+    p
+      | The purpose of this kit is to provide the means to get started with the
```
Comment (LightGuard) [inline _partials/bpm-suite-starter-kit.html.slim]: These longer sections of prose would be much better as asciidoc. See http://www.rubydoc.info/gems/slim/frames#Embedded_engines__Markdown______ for how to use the embedded engines.

File: _partials/bpm-suite-starter-kit.html.slim [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    .large-24.columns
+      ul.list-indent
+        li: a href="https://www.jboss.org/products/bpmsuite/get-started/" Get started with JBoss BPM Suite
+        li: a href="https://www.jboss.org/products/bpmsuite/developer-materials/#!hidden_keyword=JBoss%20BPMSuite" Developer materials for JBoss BPM Suite
+        li: a href="http://www.schabell.org/search/label/Tips%26Tricks" JBoss BPM Suite tips and tricks
+        li: a href="http://www.schabell.org/search/label/Video" JBoss BPM Suite videos
+        li: a href="https://www.jboss.org/products/bpmsuite/overview/#buzz" Catch all the Buzz around JBoss BPM Suite
+    p We hope this kit will help you accelerate your JBoss BPM Suite experience!
+br
+
```
Comment (LightGuard) [inline _partials/bpm-suite-starter-kit.html.slim]: Any reason for the large amount of whitespace at the end of the file?

File: _partials/bpm-suite-starter-kit.html.slim (hunk 1 above)
Comment (paulrobinson) [inline _partials/bpm-suite-starter-kit.html.slim]: +1. I Planned to walk adela through this next week when I see her in Neuchatel.

**Pull request closed without merging**